antarest/study/storage/rawstudy/model/filesystem/lazy_node.py

```diff
@@ -146,6 +147,42 @@ def get_lazy_content(
     ) -> str:
         return f"file://{self.config.path.name}"
 
+    def get_file_content(self) -> t.Tuple[bytes, str, str]:
+        """
+        Get file content
+
+        Returns:
+            file content, as bytes
+            file suffix
+            file name
+        """
+        suffix = self.config.path.suffix
+        filename = self.config.path.name
+        if self.config.archive_path:
+            return (
+                read_original_file_in_archive(self.config.archive_path, str(self.get_relative_path_inside_archive())),
+                suffix,
+                filename,
+            )
+        elif self.get_link_path().is_file():
+            with tempfile.TemporaryDirectory() as tmp_dir:
```

Comment:
As I said last time, you can use a buffer instead of using a TemporaryDirectory I think. It would make the code easier